src/runtime/interop.cs

```diff
@@ -182,6 +159,26 @@ static OriginalObjectOffsets()
         public static int ob_type;
     }
 
+
+    internal static class ManagedExceptionOffset
+    {
+        public static int ob_data { get; private set; }
+        public static int ob_dict { get; private set; }
+        public static int ob_weaklist { get; private set; }
+
+        public static int Size { get; private set; }
+
+        static ManagedExceptionOffset()
+        {
+            int baseOffset = OriginalObjectOffsets.Size + ExceptionOffset.Size();
+            ob_data = baseOffset + ManagedDataOffsets.ob_data;
+            ob_dict = baseOffset + ManagedDataOffsets.ob_dict;
+            ob_weaklist = baseOffset + ManagedDataOffsets.ob_weaklist;
```

Comment:
It could be, this offset corresponded to the `ClrPyObjectEx`. But I can't see the point, since `ExceptionOffset` is fixed, it corresponded to `PyBaseExceptionObject` in the `pyerrors.h`.